guides/development-workflow.md

```diff
@@ -122,21 +117,20 @@ general guidelines.
 1. link the Trello card to the new pull request, linking directly or by using
    the Trello
    [power up](https://help.trello.com/article/1021-integrating-trello-with-zendesk)
-1. move the Trello card into the 'Awaiting review column'
+1. move the card to indicate it's now ready for review by a peer
 
 ### Reviewing another developer's pull request
 
-- move the Trello card into the 'Under code review' column
+- move the Trello card to indicate you're now looking at it
 - find and read the Trello card's acceptance criteria
```

Comment:
I think there should always be a user need _or_ acceptance criteria (preferably both) so I've made a slightly different edit. At a basic level a user need should help the reviewer know what should be be achieveable even if there are no detailed AC. I don't agree that cards should have neither and that we might suggest that that be expected practice.

Reply:
What I'm thinking about is that sometimes the user needs exist but haven't been added to the card by the original author. I don't want to encourage an existing habit, which is "this card isn't well defined because it's missing ACs - please fix it before I can pick it up, tech lead" or similar. Adding ACs or user needs might be part of doing the work for the card and that should be ok.

Reply:
I definitely agree with that.